src/python_bindings/sip/CMakeLists.txt

```
@@ -54,6 +54,7 @@ endif()
 if(sip_helper_FOUND)
   list(APPEND rviz_BINDINGS "sip")
   set(rviz_BINDINGS "${rviz_BINDINGS}" PARENT_SCOPE)
+  get_property(rviz_sip_INCLUDE_DIRS DIRECTORY PROPERTY INCLUDE_DIRECTORIES)
```

```suggestion
  get_property(dir_includes DIRECTORY PROPERTY INCLUDE_DIRECTORIES)
  set(rviz_sip_INCLUDE_DIRS ${dir_includes} ${rviz_sip_INCLUDE_DIRS})
```

this combines both the manually crafted list from above as well as the one that is set for the directory (which probably adds the build directory where rviz_export.h is found)